### DISCUSSION TITLE
[FR] Implement an alternative to old machines not compatible with flutter

### DISCUSSION THREAD

**Discussion by FurtadoPires** [rustdesk/rustdesk#8766] [General]
I noticed that rustdesk can't be used in old machines because flutter doesn't work with >=opengl2.1, where programs like team viewer and anydesk work, because they're using another api for their interface.

Since flutter is the only thing preventing rustdesk to be used in those machines, I would like to know if it's possible to have an alternative mode / package for those machines. 

Even a CLI option would be good if it's possible, as long as it could work with other functions.

**Comment by deep-soft** [comment]
for Windows:
https://github.com/rustdesk/rustdesk/releases/download/1.2.6/rustdesk-1.2.6-x86-sciter.exe

**Reply by FurtadoPires** [reply]
> for Windows

Only windows right? In my case the machine is running Linux with wayland. 
Also I think windows are still covered by flutter due to dx drivers, it's really linux (and mac maybe?) that are affected in this case.

**Reply by rustdesk** [reply]
1.2.7

**Reply by deep-soft** [reply]
sorry, forgot the nightly is after 1.2.6
also for linux (deb)
https://github.com/rustdesk/rustdesk/releases/download/nightly/rustdesk-1.2.7-x86_64-sciter.deb
more here:
https://github.com/rustdesk/rustdesk/releases/tag/nightly

**Reply by FurtadoPires** [reply]
> 1.2.7

Just tested with the flatpak version and it's working!

Thank you @rustdesk and the developer team
